### DISCUSSION TITLE
Office_S_Tips/excel/2021/12/14/Kilka_arkuszy_kalkulacyjnych

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#31] [Komentarze]
# Office_S_Tips/excel/2021/12/14/Kilka_arkuszy_kalkulacyjnych

1. Czy święta wypadają w sobotę   2. Planowanie urlopów  

https://andrzejq.github.io/Office_S_Tips/excel/2021/12/14/Kilka_arkuszy_kalkulacyjnych.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
